app/views/profiles/timeline_partials/_comment.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="row commented">
+  <hr>
+  <div class="col-sm-2">
+    <div class="thumbnail">
+      <img src="images/user_silhouette_generic.gif">
+    </div>
+  </div> <!-- col-md-2 end -->
+
+  <div class="col-md-7 poster">
+    <%= comment.user.first_name %><span class="grey-text">  <%= comment.created_at.strftime('Said on %A %D') %></span>
```

Comment:
It appears that you do not use the user's last name anywhere.

CTRL+F for "user.la" returns nothing.

If you do decide that you want to show a user's last name, I recommend the virtual attribute.

The one I had in the user's model was like this:
def full_name
    user.first_name + " " + user.last_name
end